packages/repo-tools/src/commands/backend-lint/backend-lint.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import fs from 'fs-extra';
+import { Project } from 'ts-morph';
+import { resolvePackagePaths } from '../../lib/paths';
+
+const project = new Project({
+  tsConfigFilePath: 'tsconfig.json',
+});
+
+export async function lint(paths: string[]) {
+  const pkgs = (await resolvePackagePaths()).filter(pkg => {
```

Comment:
Seeing as we don't filter at all it's probably cleaner to use `PackageGraph.listTargetPackages()` directly. That avoids the need to load in the `package.json` files again.